build.gradle

```
@@ -193,6 +193,13 @@ subprojects {
     configurations {
         integrationTestImplementation.extendsFrom testImplementation
         integrationTestRuntime.extendsFrom testRuntimeClasspath
+        
+        // Force slf4j version to 1.7.36 for Kafka compatibility
+        all {
```

To follow Gradle best practices regarding configuration avoidance, could we use `configureEach`? 
```suggestion
        configureEach {
```